src/_macosx.m

```diff
@@ -252,6 +252,20 @@ - (BOOL)acceptsFirstResponder;
 
 /* ---------------------------- Python classes ---------------------------- */
 
+// Acquire the GIL, call a method with no args, discarding the result and
+// printing any exception.
+void gil_call_method(PyObject* obj, const char* name)
```

Comment:
This should be `static` as it doesn't need to be called elsewhere.

Reply:
sure.